rails/app/views/home/_card.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="card">
   <div class="logo">
-    <%= image_tag 'logocombo', alt: 'Terrastories' %>
+    <%= image_tag 'logocombo.svg', alt: 'Terrastories' %>
```

Comment:
Filename needs to match, otherwise it causes a missed lookup in the asset pipeline when running in production.